app/src/main/java/app/revanced/integrations/patches/spoof/requests/StoryBoardRendererRequester.java

```diff
@@ -31,10 +31,12 @@ private StoryBoardRendererRequester() {
     }
 
     // TODO: Find a way to increase the quality of SeekBar thumbnail previews
-    public static void fetchStoryboardsRenderer(@NonNull String videoId) {
-        ReVancedUtils.verifyOffMainThread();
-
+    @Nullable
+    public static String fetchStoryboardsRenderer(@NonNull String videoId) {
         try {
+            // Calling code is not in a try catch block and an exception will take down the app.
+            // Instead verify in this block.
+            ReVancedUtils.verifyOffMainThread();
             final byte[] innerTubeBody = String.format(INNER_TUBE_BODY, videoId).getBytes(StandardCharsets.UTF_8);
```

Comment:
Good point, removing the comment as it is not really relevant for the patch, but only acted as an explanation for this PR
```suggestion
            ReVancedUtils.verifyOffMainThread();

            final byte[] innerTubeBody = String.format(INNER_TUBE_BODY, videoId).getBytes(StandardCharsets.UTF_8);
```